R-package/DESCRIPTION

```diff
@@ -0,0 +1,28 @@
+Package: RGF
+Type: Package
+Title: Regularized Greedy Forest
+Version: 1.0.4
+Date: 2018-07-22
+Authors@R: c( person("Lampros", "Mouselimis", email = "mouselimislampros@gmail.com", role = c("aut", "cre")), person("Ryosuke", "Fukatani", role = "cph", comment = "Author of the python wrapper of the 'Regularized Greedy Forest' machine learning algorithm"), person("Tong", "Zhang", role = "cph", comment = "Author of the 'Regularized Greedy Forest' and of the Multi-core implementation of Regularized Greedy Forest machine learning algorithm"), person("Rie", "Johnson", role = "cph", comment = "Author of the 'Regularized Greedy Forest' machine learning algorithm") )
+Maintainer: Lampros Mouselimis <mouselimislampros@gmail.com>
+BugReports: https://github.com/mlampros/RGF/issues
+URL: https://github.com/mlampros/RGF
+Description: Regularized Greedy Forest wrapper of the 'Regularized Greedy Forest' <https://github.com/fukatani/rgf_python> 'python' package, which also includes a Multi-core implementation (FastRGF) <https://github.com/baidu/fast_rgf>.
+License: MIT + file LICENSE
+SystemRequirements: Python (2.7 or >= 3.4), rgf_python, scikit-learn (>= 0.18.0), scipy ( >= 1.0.0), numpy ( >= 1.14.0). Detailed installation instructions for each operating system can be found in the README file.
```

Comment:
>scipy ( >= 1.0.0), numpy ( >= 1.14.0).

Can you please explain this?

Reply:
The RGF R package includes the *mat_2scipy_sparse* and *TO_scipy_sparse* functions. For these two functions to work properly, scipy and numpy are the requirements.

Reply:
Yeah, sure! rgf_python requires these packages too, because `scikit-learn` requires them =)
I'm wondering about such strange (very fresh) **versions**.